controllers/user.js

```diff
@@ -32,6 +34,25 @@ async function add(req, res) {
 async function verify(req, res) {
   const verifiedEmailHash = await DB.verifyEmailHash(req.query.token);
 
+  const unsafeBreachesForEmail = HIBP.getUnsafeBreachesForEmail(
+    sha1(verifiedEmailHash.email),
+    req.app.locals.breaches
+  );
+
+  const unsubscribeUrl = EmailUtils.unsubscribeUrl(verifiedEmailHash);
+
+  await EmailUtils.sendEmail(
+    verifiedEmailHash.email,
+    "Your Firefox Monitor report",
+    "welcome",
+    {
+      email: verifiedEmailHash.email,
+      date: new Date(Date.now()),
```

Comment:
This feels "off".
`new Date()` should give you the current date (as a Date object).
`Date.now()` should give you the current date (as an integer).

Using both feels a bit unnecessary.